src/services/storage-options/restApiService.js

```diff
@@ -82,32 +91,28 @@ export default class RestApiService extends StorageInterface {
   }
 
   async getConfig() {
-    const response = await axios.get(
-      `http://localhost:5000/v1/app/org/f352ae63-11fc-4dbe-bab1-72561aa25fca/config/5e0f71ff-987d-4240-85eb-df6adf568c31`
+    const response = await this.#api.get(
+      `app/org/f352ae63-11fc-4dbe-bab1-72561aa25fca/config/5e0f71ff-987d-4240-85eb-df6adf568c31`
     );
     return response.data.config;
   }
 
   async getAttachment(attachmentId) {
-    const response = await axios.get(
-      `http://localhost:5000/v1/attachments/${attachmentId}/object`
-    );
+    const response = await this.#api.get(`attachments/${attachmentId}/object`);
     console.log(response.data);
     return response.data;
   }
 
   async updateConfig(config) {
-    const response = await axios.put(
-      `http://localhost:5000/v1/app/org/f352ae63-11fc-4dbe-bab1-72561aa25fca/config/5e0f71ff-987d-4240-85eb-df6adf568c31`,
+    const response = await this.#api.put(
+      `app/org/f352ae63-11fc-4dbe-bab1-72561aa25fca/config/5e0f71ff-987d-4240-85eb-df6adf568c31`,
       { config }
     );
     return response.data.config;
   }
 
   async getCredentials() {
```

Comment:
this is where you should check if a cookie is set.

if we are running in the browser then check for cookies first and return the credential like:
```
return {
  yatt: [
    {
      type: "cookie",
      ...
    }
  ]
};
```

And extract the user data from the JWT inside the cookie.

Reply:
if no cookie is set then fall back to the API endpoitn here